### ISSUE TITLE
Scudlee Files Out-of-Date? How to force update?

### ISSUE THREAD

**Issue by Neipas09** [ZeroQI/Hama.bundle#548]
The XML files from ScudLee haven't been updated for over a month, and i see he did an update like 4 hours ago.
Is there a way to force HAMA to re-download all the XML files from ScudLee so it's up-to-date?

**Comment by ZeroQI** [comment]
https://github.com/ZeroQI/Hama.bundle/blob/master/Contents/Code/AnimeLists.py#L15
https://github.com/Anime-Lists/anime-lists/ indeed show an update 8 hours ago in:
- https://raw.githubusercontent.com/Anime-Lists/anime-lists/master/anime-list-master.xml

You can delete the file in gant data folder > anime-lists folder but Plex might hold a chache still, so if you custom search for a series name called 'clear-cache' it will reload from the site without plex cache and without disk cache

**Comment by Neipas09** [comment]
What is "gant data folder" ???

**Comment by ZeroQI** [comment]
Agent*

**Comment by Neipas09** [comment]
I got the clear-cache to work.
![image](https://user-images.githubusercontent.com/58871039/233852306-09ec8c57-550a-4ac1-8b26-984a891fb18e.png)

But for whatever reason, the anime-list's aren't downloading.
I deleted the folder [C:\Users\PlexServer\AppData\Local\Plex Media Server\Plug-in Support\Data\com.plexapp.agents.hama] thinking it would fix my issue, but the anime-list files aren't re-downloading.
